R/phone-number-provider-all.R

```diff
@@ -114,19 +114,16 @@ phone_number_formats_el_gr = c(
   '+302#########'
 )
 
-phone_number_formats_dk_dk = c(
-  '+45(0)##########',
-  '+45(0)#### ######',
-  '+45 (0) #### ######',
-  '+45(0) #########',
-  '+45(0)#### #####',
-  '0##########',
-  '0#########',
-  '0#### ######',
-  '0#### #####',
-  '(0####) ######',
-  '(0####) #####'
-)
+phone_number_formats_da_dk = c(
```

Comment:
@MartinMSPedersen  You removed some of the possible formats. curious if you just now the possible formats, or if you have a reference on the web?

Reply:
Phonenumbers in denmark is always 8 digits.
The country-code is 45 and it is something prefixed.
And there is never a zero between the country-code and the number.

Regards
Martin

Reply:
Thanks for the package. Yes, I know the danish number. I am danish and have lived all my life in Denmark.
You can also check this page:
https://en.wikipedia.org/wiki/Telephone_numbers_in_Denmark

Regards
Martin

Reply:
Okay. Thanks very much.